data/movie/The Peanuts Movie (2015)/collectors-edition/disc01-summary.txt

```diff
@@ -0,0 +1,451 @@
+
+Name: Official Trailer 2
+Type: Trailer
+Source file name: 02548.m2ts
+Duration: 0:02:26
+Size: 447.1 MB
+Segment count: 1
+Segment map: 2548
+File name: Trailer 2.mkv
+
+Name: Teaser Trailer 1
+Type: Trailer
+Source file name: 02547.m2ts
+Duration: 0:00:56
+Size: 172.1 MB
+Segment count: 1
+Segment map: 2547
+File name: Teaser 1.mkv
+
+Name: Teaser Trailer 2
+Type: Trailer
+Source file name: 02566.m2ts
+Duration: 0:01:36
+Size: 294.9 MB
+Segment count: 1
+Segment map: 2566
+File name: Teaser 2.mkv
+
+Name: Official Holiday Trailer
+Type: Trailer
+Source file name: 02567.m2ts
+Duration: 0:01:41
+Size: 309.9 MB
+Segment count: 1
+Segment map: 2567
+File name: Holiday Trailer.mkv
+
+Name: Snoopy's Sibling Statue
+Type: Extra
+Source file name: 02402.m2ts
+Duration: 0:01:55
+Size: 360.3 MB
+Segment count: 1
+Segment map: 2402
+File name: Snoopy's Sibling Statue.mkv
+
+Name: Learn to Draw Snoopy
+Type: Extra
+Description: Presented by Steve Martino, the director of the film
+Source file name: 02403.m2ts
+Duration: 0:04:13
+Size: 800.1 MB
+Segment count: 1
+Segment map: 2403
+File name: Learn to Draw Snoopy.mkv
+
+Name: Learn to Draw Woodstock
+Type: Extra
+Description: Presented by Steve Martino, the director of the film
+Source file name: 02404.m2ts
+Duration: 0:03:04
+Size: 579.9 MB
+Segment count: 1
+Segment map: 2404
+File name: Learn to Draw Woodstock.mkv
+
+Name: Learn to Draw Charlie Brown
+Type: Extra
+Description: Presented by Steve Martino, the director of the film
+Source file name: 02519.m2ts
+Duration: 0:04:02
+Size: 771.8 MB
+Segment count: 1
+Segment map: 2519
+File name: Learn to Draw Charlie Brown.mkv
+
+Name: Behind the Scenes of "Better When I'm Dancin'"
+Type: Extra
+Source file name: 02520.m2ts
+Duration: 0:02:53
+Size: 538.0 MB
+Segment count: 1
+Segment map: 2520
+File name: Better When I'm Dancin' behind the scenes.mkv
+
+Name: "Better When I'm Dancin'" Meghan Trainor lyric video
+Type: Extra
+Source file name: 02535.m2ts
+Duration: 0:03:01
+Size: 560.5 MB
+Segment count: 1
+Segment map: 2535
+File name: Better When I'm Dancin' animated.mkv
+
+Name: Get Down with Snoopy and Woodstock Music Video
+Type: Extra
+Source file name: 02534.m2ts
+Duration: 0:02:17
+Size: 421.7 MB
+Segment count: 1
+Segment map: 2534
+File name: Get Down With Snoopy and Woodstock.mkv
+
+Name: "Better When I'm Dancin'" Meghan Trainor music video
+Type: Extra
+Source file name: 02536.m2ts
+Duration: 0:03:08
+Size: 580.2 MB
+Segment count: 1
+Segment map: 2536
+File name: Better When I'm Dancin' live-action.mkv
+
+Name: Pre-top-menu sequence with partial Ice Age short
+Type: Extra
+Source file name: 02560.m2ts
+Duration: 0:02:32
+Size: 372.9 MB
+Segment count: 1
+Segment map: 2560
+File name: Disc opening sequence.mkv
+
+Name: Studio sequences & title card animation without movie name
+Type: Extra
+Source file name: 02577.m2ts
+Duration: 0:01:28
+Size: 427.6 MB
+Segment count: 1
+Segment map: 2577
+File name: Movie opening sequence.mkv
```

Comment:
I also wouldn't consider this an extra. I would like to hear more about your intention in labeling this too